<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgorithmRunnerItem</class>
 <widget class="QWidget" name="AlgorithmRunnerItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>486</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="algFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
border-image: none;
border: 1px solid #E9E9E9;
}

#settingsButton {
border: 0px;
}

QPushButton:hover, #settingsButton:hover {
border-image: none;
border: 1px solid #C9C9C9;
}

QPushButton:pressed, #settingsButton:pressed {
border-image: none;
background-color: #E9E9E9;
border: 1px solid #C9C9C9;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="settingsButton">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string notr="true">Left click: show/hide algorithm settings,
Right click: display plugin documentation</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/tulip/gui/icons/16/settings.png</normaloff>:/tulip/gui/icons/16/settings.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/tulip/gui/icons/22/start.png</normaloff>:/tulip/gui/icons/22/start.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="favoriteCheck">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add to Favorites</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
spacing: 0px;
}
QCheckBox::indicator {
width: 16px;
height: 16px;
}
QCheckBox::indicator:unchecked {
image: url(:/tulip/graphperspective/icons/16/favorite-empty.png);
}
QCheckBox::indicator:unchecked:hover {
image: url(:/tulip/graphperspective/icons/16/favorite-hover.png);
}
QCheckBox::indicator:unchecked:pressed {
image: url(:/tulip/graphperspective/icons/16/favorite.png);
}
QCheckBox::indicator:checked {
image: url(:/tulip/graphperspective/icons/16/favorite.png);
}
QCheckBox::indicator:checked:hover {
image: url(:/tulip/graphperspective/icons/16/favorite-hover.png);
}
QCheckBox::indicator:checked:pressed {
image: url(:/tulip/graphperspective/icons/16/favorite-empty.png);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="parameters">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>27</number>
     </attribute>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>playButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>AlgorithmRunnerItem</receiver>
   <slot>run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settingsButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>parameters</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settingsButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>AlgorithmRunnerItem</receiver>
   <slot>initModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>favorized(bool)</signal>
  <slot>run()</slot>
  <slot>initModel()</slot>
 </slots>
</ui>
